src/Akka.HealthCheck.Cluster.Tests/Akka.HealthCheck.Cluster.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Akka.TestKit.Xunit" Version="$(AkkaVersion)" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
```

Comment:
Ideally we should upgrade to the latest here and reference this from `common.props` / `Directory.Build.props` but we can do that later